apps/tauri/Makefile

```
@@ -0,0 +1,54 @@
+	@echo "🚀 Starting Tauri in development mode..."
+	npm run tauri dev
+
+build:
+	@echo "📦 Building Tauri app (production)..."
+	npx tauri build
+
+build-dev:
+	@echo "🔨 Building local dev version with git hash..."
+	@../../scripts/build-dev.sh
+
+install:
```

checkmake (0.2.2)

[warning] Target body for "install" exceeds allowed length of 5 (14).

(maxbodylength)